File: array/1252_cells_with_odd_values_in_a_matrix/1252_cells_with_odd_values_in_a_matrix.py
class Solution:
    def oddCells(self, n: int, m: int, indices: List[List[int]]) -> int:
        rows = [0] * n
        cols = [0] * m
        for x, y in indices:
            rows[x] += 1
            cols[y] += 1
        return sum((rows[x] + cols[y]) % 2 == 1 for x in range(n) for y in range(m))

File: array/1252_cells_with_odd_values_in_a_matrix/1252_cells_with_odd_values_in_a_matrix.go
package main

func oddCells(n int, m int, indices [][]int) int {
	row := make([]int, n)
	col := make([]int, m)

	for _, indice := range indices {
		row[indice[0]]++
		col[indice[1]]++
	}

	res := 0
	for _, v := range row {
		for _, vv := range col {
			if (v+vv)%2 == 1 {
				res++
			}
		}
	}

	return res
}
